src/gsy_e/models/strategy/load_hours.py

```diff
@@ -50,12 +50,123 @@
 BalancingRatio = namedtuple("BalancingRatio", ("demand", "supply"))
 
 
+class LoadHoursEnergyParameters:
+    """Energy related parameters of the Load Hours Strategy."""
+    def __init__(self, avg_power_W, hrs_per_day=None, hrs_of_day=None, area=None):
```

Comment:
It seems you are expecting an area object in the constructor here, is there a particular reason for that? I did not find any initializations of this class apart from within the `LoadHoursStrategy`, and we're not passing an area object there

Reply:
Indeed this member is populated during the activate method, therefore no need for the constructor to also include it. Removing it from here, thanks!

Reply:
Might be on a later commit than this one, but you can find it in the method `LoadHoursEnergyParameters.event_activate_energy`